src/net/sourceforge/kolmafia/request/FightRequest.java

```diff
@@ -9124,7 +9126,7 @@ private static void payActionCost(final String responseText) {
           // Your mind fills with it essence. You... know it. With a capital K.
           if (responseText.contains("Your mind fills") || jiggleSuccess) {
             Preferences.increment("_jiggleCream", 1);
-            Preferences.setString("_jiggleCreamedMonster", monsterName);
+            TrackManager.trackCurrentMonster(Tracker.CREAM_JIGGLE);
```

Comment:
Why `trackCurrentMonster` here and `trackMonster(monsterName...` later?

Reply:
I think the standard is that it's `trackCurrentMonster` for skills and `trackMonster` for items.

Now as to /why/ it's like that, that's another question, but I mimicked whatever banish did.